playground/frontend/lib/pages/embedded_playground/components/embedded_actions.dart

```diff
@@ -58,16 +59,29 @@ class EmbeddedActions extends StatelessWidget {
   void _openStandalonePlayground(PlaygroundController controller) {
     // The empty list forces the parsing of EmptyExampleLoadingDescriptor
     // and prevents the glimpse of the default catalog example.
+
+    final descriptor = controller.getLoadingDescriptor();
+    final descriptors = descriptor.descriptors;
+    final urlDescriptors = descriptors.where((d) => d.canBePassedInUrl);
+    final jsDescriptors = descriptors.where((d) => !d.canBePassedInUrl);
+
+    final json = jsonEncode(
+      urlDescriptors.map((d) => d.toJson()).toList(growable: false),
+    );
     final window = html.window.open(
-      '/?$kExamplesParam=[]&$kSdkParam=${controller.sdk?.id}',
+      '/?$kExamplesParam=$json&$kSdkParam=${controller.sdk?.id}',
       '',
     );
 
-    javaScriptPostMessageRepeated(
-      window,
-      SetContentMessage(
-        descriptor: controller.getLoadingDescriptor(),
-      ),
-    );
+    if (jsDescriptors.isNotEmpty) {
+      javaScriptPostMessageRepeated(
+        window,
+        SetContentMessage(
+          descriptor: descriptor.copyWith(
```

Comment:
I don't think we should copy it here.

1. `descriptor` may contain `initialSdk`. At best it is redundant since we pass it in the URL. At worst it can switch the SDK if the user changed it after loading from URL and before the message is received.
2. `descriptor` may contain `lazyLoadDescriptors`. They have no effect in this message currently but may get in the way if we ever stop ignoring them.

Reply:
Thanks, fixed